### DISCUSSION TITLE
MSS810 supported?

### DISCUSSION THREAD

**Discussion by Sinnedterklaas** [krahabb/meross_lan#381] [Q&A]
It isn't in the official supported list but does anyone knows if it works with home assistant? I already have the MSS315 smart switch and MSL120 light bulb and those are quite stable for me so I was interested in the MSS810.

**Comment by krahabb** [comment]
I think it should work since the various meross appliances are very similar between each other and so highly compatible on the protocol side
Especially switches, which have the 'most long standing api' for the family.

**Comment by Sinnedterklaas** [comment]
I can confirm the MSS810 works perfectly. It could be added to the list of supported devices. 

Great work!
